docs/configuration.md

```diff
@@ -233,6 +274,7 @@ spec:
 | clusterAgent.config.externalMetrics.port | If specified configures the metricsProvider external metrics service port. |
 | clusterAgent.config.externalMetrics.useDatadogMetrics | Enable usage of DatadogMetrics CRD (allow to scale on arbitrary queries). |
 | clusterAgent.config.externalMetrics.wpaController | Enable informer and controller of the watermark pod autoscaler. NOTE: The WatermarkPodAutoscaler controller needs to be installed. See also: https://github.com/DataDog/watermarkpodautoscaler. |
+| clusterAgent.config.healthPort | HealthPort of the agent container for internal liveness probe. Must be the same as the Liness/Readiness probes. |
```

Comment:
```suggestion
| clusterAgent.config.healthPort | HealthPort of the Agent container for internal liveness probe. Must be the same as the Liveness/Readiness probes. |
```

Should it be "liveness" instead of liness?